### PR TITLE
Doing the Learn front page and section styling

### PR DESCRIPTION
**Pull request by kmurphypolygon**
I added the web dev Alan's changes to this and removed some of my efforts from before to keep it low key and subtle.

We could probable push this now. There might be some snagging.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -118,14 +136,14 @@ hide:
...
+						<a href="learn/polygon-protocols" class="flex-card-item w-inline-block">
 							<div class="product-list-item-header">
-								<div class="feature-card-heading">Architecture</div>
+								<div class="feature-card-heading">Polygon protocols</div>
 								<div class="arrow-embed w-embed"><svg xmlns="http://www.w3.org/2000/svg" display="block" width="100%" height="auto" viewbox="0 0 16 17" fill="none">
 										<path d="M9.98805 5.70133L3.41071 12.2787L4.58922 13.4572L11.1666 6.87976V12.2013H12.8333V4.03467H4.66665V5.70133H9.98805Z" fill="currentColor"></path>
 									</svg></div>
 							</div>
-							<p class="feature-paragraph">The Polygon protocol that’s best for you. A guide and decision matrix.</p>
+							<p class="feature-paragraph">The Polygon protocol that’s best for you. A guide and decision matrix designed to empower users to navigate the evolving world of decentralized.</p>
```
Comment (hsutaiyu) [inline docs/index.md]: "evolving world of decentralization"?

File: docs/learn/index.md [hunk 2]
```diff
@@ -1,23 +1,58 @@
...
+            background-color: #181818 !important;
+      }
+   }
+</style>
+
+<div class="section-wrapper product-section-head">
+      <div class="hero-image"><img src="../img/learn/learn.svg" loading="lazy" class="hero-image" style="width: 40%; float: right;"></div>
+   <div class="hero-left">
+      <h1 class="hero-heading">Learn</h1>
+      <p class="hero-subtext">The Polygon Knowledge Layer consists of two parts. Firstly, documents that developers need in order to build with Polygon protocols. Secondly, resources necessary for learning about Polygon technologies.</p>
```
Comment (hsutaiyu) [inline docs/learn/index.md]: "Firstly, _tech docs that assist developers_ to build with..."?

**Review by hsutaiyu — APPROVED** [review]
Approving with minor suggestions.

File: docs/index.md (hunk 1 above)
Comment (kmurphypolygon) [reply docs/index.md]: DONE in a commit

File: docs/learn/index.md (hunk 2 above)
Comment (kmurphypolygon) [reply docs/learn/index.md]: Done in a commit.

**Pull request merged**